tests/console/ansible.pm

```diff
@@ -0,0 +1,164 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Package: ansible
+# Summary: Test Ansible and its components
+#   1. System setup
+#   2. Ansible basics
+#   3. Ansible Galaxy
+#   4. Ansible playbook testing
+#   5. Ansible playbook execution
+#   6. Ansible Vault
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use warnings;
+use base "consoletest";
+use strict;
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils qw(zypper_call random_string systemctl);
+use version_utils 'is_sle';
+use registration qw(add_suseconnect_product get_addon_fullname);
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # 1. System setup
+
+    # Instal Ansible from PIPY
+    #assert_script_run 'pip3 install ansible yamllint', 300;
+
+    if is_sle {
+        add_suseconnect_product(get_addon_fullname('phub'));
+    }
+
+    # Install ansible and ansible-test
+    # Install python3-yamllint needed for ansible-test
+    # Install git needed for ansible-galaxy
+    zypper_call 'in ansible python3-yamllint git-core';
+
+    # Enable sshd
+    systemctl 'start sshd';
```

Comment:
Do you mean `systemctl 'enable --now sshd';` `start` will start the service but it wont same as enable.

Reply:
No. It's just wrong comment :smile: